app/views/support_interface/application_forms/application_choices/confirm_reinstate_offer.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :browser_title, title_with_error_prefix('Reinstate offer', @declined_course_choice.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(application_form_id: params[:application_form_id]), 'Back to application') %>
+
+<%= form_with(
+  model: @declined_course_choice,
+  url: support_interface_application_form_reinstate_offer_path(application_form_id: params[:application_form_id], application_choice_id: params[:application_choice_id]),
+  method: :patch
+  )do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_error_summary %>
+      <h1 class="govuk-heading-l">
+        Are you sure you want to reinstate the offer for this course choice?
+      </h1>
+
+      <h3 class="govuk-heading-s govuk-!-margin-bottom-2">An offer can only be reinstated if:</h3>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>it was declined within 5 working days of the request to reinstate and,</li>
```

Comment:
Are there common variables we can use for these working days values?

Reply:
There aren't at the moment.

Maybe we could add something like `EDITABLE_DAYS = 5` to the application form model then do 

`        <li>it was declined within <%=ApplicationForm::EDITABLE_DAYS %> working days of the request to reinstate and,</li>
`